Assets/Scripts/SceneManager.cs

```diff
@@ -5,10 +5,28 @@
 public class SceneManager : MonoBehaviour
 {
     public int nextSceneNum;
-
-    public void loadScene(int nextSceneNum)
+    public Vector3 position;
+    public string UIText;
+    public GameObject UI;
+    public GameObject player;
+    public GameObject sceneManage;
+    public GameObject gameManager;
+    public void loadScene(int nextSceneNum, GameObject door)
     {
-        // animation to go through door - > black screen until other side
-        UnityEngine.SceneManagement.SceneManager.LoadScene(nextSceneNum);
+        if (nextSceneNum == 0)
+        {
+            Destroy(UI);
+            Destroy(player);
+            Destroy(sceneManage);
+            Destroy(gameManager);
```

Comment:
Does Unity not automatically destroy when loading new scenes

Reply:
No, my next task is to actually MOVE the game objects rather than using dont destroy and then destroy when loading in the scene that created the objects. Or in my case, the bed room scene 0.